bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -393,35 +231,34 @@ logging.info("SNR and bestNR maxima calculated.")
 
 # Output details of loudest offsouce triggers, sorted by BestNR
 offsource_trigs = []
-sorted_trigs = ppu.sort_trigs(trial_dict, trigs, slide_dict, segment_dict)
+sorted_trigs = ppu.sort_trigs(trial_dict, trig_data, slide_dict, segment_dict)
 for slide_id in slide_dict:
-    offsource_trigs.extend(zip(trig_bestnr[slide_id], sorted_trigs[slide_id]))
+    offsource_trigs.extend(zip(trig_bestnr[slide_id],
+                               get_trig_dictionary(sorted_trigs[slide_id])))
+
 offsource_trigs.sort(key=lambda element: element[0])
 offsource_trigs.reverse()
 
 # Median and max values of SNR and BestNR
-_, median_snr, _ = ppu.max_median_stat(slide_dict, time_veto_max_snr, trig_snr, total_trials)
+_, median_snr, _ = ppu.max_median_stat(slide_dict, time_veto_max_snr,
+                                       trig_snr, total_trials)
 max_bestnr, median_bestnr, full_time_veto_max_bestnr =\
-    ppu.max_median_stat(slide_dict, time_veto_max_bestnr, trig_bestnr, total_trials)
+    ppu.max_median_stat(slide_dict, time_veto_max_bestnr, trig_bestnr,
+                        total_trials)
 
 if lofft_outfile or lofft_h5_outfile:
     # td: table data
     td = []
 
-    # NOTE: Rather than changing the xml table structures, KAGRA piggy backs
-    # on the TAMA column (t). The h2 column is left unused.
-    # TAMA and Hanford2 are therefore no longer supported.
-    ifo_att = {'G1':'g', 'H1':'h1', 'K1': 't', 'L1':'l', 'V1':'v'}
-
     # Gather properties of the loudest offsource triggers
     for i in range(min(len(offsource_trigs), opts.num_loudest_off_trigs)):
         bestnr = offsource_trigs[i][0]
         trig = offsource_trigs[i][1]
-        trig_slide_id = int(trig.time_slide_id)
+        trig_slide_id = int(trig['time_slide_id'])
```

Comment:
Assuming you change previous code to use `event_id`s, you could loop through those directly. Also the time slide ID is stored under `file['network/slide_id']`